lms/templates/instructor/instructor_dashboard_2/student_admin.html

```diff
@@ -1,6 +1,13 @@
 <%! from django.utils.translation import ugettext as _ %>
 <%page args="section_data"/>
 
+<script>
```

Comment:
Although I think we should be putting this js file somewhere in `lms/static/js`
@caesar2164 right?